profile/README.md

```diff
@@ -1,4 +1,4 @@
-🦍 A helpful toolkit for building HTTP-based applications with the [Go programming language](https://go.dev/).
```

Comment:
Out of curiosity - why the decision to remove the link?

Reply:
I figured that everyone knows how to find the go programming website by now